samples/interactive-tutorials/product/add_fulfillment_places.py

```diff
@@ -64,7 +62,7 @@ def add_fulfillment_places(product_name: str, timestamp, place_id):
     # This is a long running operation and its result is not immediately present with get operations,
     # thus we simulate wait with sleep method.
     print("---add fulfillment places, wait 40 seconds :---")
```

Comment:
Update comment to match (now 70 seconds)

Reply:
Also, why 70 seconds? Why not pick 60 or 90 seconds that would look more standard?

Reply:
+1 for 90 seconds to be more stable for testing, and fix the doc above. Also should we use the same wait tie for remove/set APIs as well?

Reply:
increased to 90